src/Core/Managed/Shared/Channel/IDeepCloneable.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Microsoft.ApplicationInsights.Channel
```

Comment:
We'd like to keep the root namespace empty so the only thing there is `TelemetryClient`. Avoiding confusions on how to use Application Insights. Maybe place alongside other telemetry items in `Microsoft.ApplicationInsights.DataContracts`?